src/server/app-router.ts

```diff
@@ -165,16 +177,21 @@ export function getRouteHandlers(args?: RouteHandlerArgs) {
             const data = await response.json()
 
             const accessToken = data.access_token
-            const path = args?.postLoginRedirectPathFn ? args.postLoginRedirectPathFn(req) : "/"
-            if (!path) {
-                console.log("postLoginPathFn returned undefined")
+
+            // If we have a return_to_path cookie, we'll use that
+            // Otherwise, we'll use the postLoginRedirectPathFn
+            const returnToPathFromCookie = req.cookies.get(RETURN_TO_PATH_COOKIE_NAME)?.value;
+            const returnToPath = returnToPathFromCookie ?? (args?.postLoginRedirectPathFn ? args.postLoginRedirectPathFn(req) : "/")
+            if (!returnToPath) {
+                console.log("postLoginRedirectPathFn returned undefined")
```

Comment:
Do we want to log here in production? I saw in most of the other frontend repos there is an eslint rule to error on console.log, just want to make sure this is meant to be here

Reply:
This should probably be console.error, that's a good point. I do want to give a message to the user, but log is probably wrong, I'll fix it.